frontend_app_file/src/container/PopupCreateFile.jsx

```diff
@@ -129,9 +131,9 @@ class PopupCreateFile extends React.Component {
     })
   }
 
-  filesAlreadyInListFilter = (newFile, listToFilter) => listToFilter.some(stateFile => stateFile.name === newFile.name)
+  hasFilesAlreadyInList = (newFile, listToFilter) => listToFilter.some(stateFile => stateFile.name === newFile.name)
```

Comment:
rename the parameters to be independent of the context of the function usage
also, the function only search one file in the list so `File` should not bu plural in the function name